env.example

```diff
@@ -3,3 +3,4 @@ GOOGLE_CLIENT_EMAIL="example_email@developer.gserviceaccount.com"
 HUBOT_GITTER2_TOKEN=GitterSampleToken
 GIST_USERNAME="<BOT_USERNAME>"
 GIST_PASSWORD="<BOT_PASSWORD>"
+GITHUB_TOKEN="GitHubToken"
```

Comment:
@Mikanebu think you should say couple of words about GITHUB_TOKEN in README. Eg paste link how/where to get it, or at least list it with other envs